src/components/header/index.tsx

```diff
@@ -27,9 +49,20 @@ const Header = (props: HeaderProps) => {
         top: 0,
       }}
     >
+      {/* TODO: similarly refactor generic info alert messages into a centralized util file
+        and corresponding piece of global state; use this to set the `top` attribute passed to error alerts below.
+      */}
       {isAppRunningInStaging() ? <StagingBanner /> : <BetaBanner />}
-      <FetchError />
-      <InvalidJsonErrorBanner />
+
+      {errorTypesToDisplay.map((error: [string, string], i: number) => {
+        return (
+          <GenericErrorBanner
+            errorCategory={error[0]}
+            errorType={error[1]}
+            top={95 + 110 * i}
```

Comment:
I think this will work, but you'd probably be better off abstracting it into a container that has vertical flex, and map the errors in that container. That way you don't have to mess around with manually setting the `top` CSS offset to avoid overlap, you can let flex handle that for you.

Reply:
That makes far more sense. Will fix. Thanks!